esindexer/reindex.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func mustDialPostgres() *pgxpool.Pool {
-	connConfig, err := pgxpool.ParseConfig(config.Cfg.WriteDbUrl)
+	connConfig, err := pgxpool.ParseConfig(config.Cfg.ReadDbUrl)
```

Comment:
wait why do you want to check this in? doesn't the indexer need the write db url?